src/js/components/molecules/tabs/tabs.ts

```diff
@@ -73,16 +71,15 @@ export class Tabs extends Vue implements TabsProps {
 
     mounted() {
         this.activeTab = this.defaultTab;
-        this.viewportUtil.addResizeHandler(this.positionIndicator);
 
+        this.setActiveTab();
+        this.viewportUtil.addResizeHandler(this.positionIndicator);
         this.$refs.tabList.addEventListener('scroll', this.scrollCorrection);
 
         if (BrowserCapabilities.supportsTouch) {
             this.$refs.tabList.addEventListener('touchmove', this.scrollCorrection);
         }
     }
 
-    beforeDestroy() {
-        this.viewportUtil.removeResizeHandler(this.positionIndicator);
-    }
+    beforeDestroy = () => this.viewportUtil.removeResizeHandler(this.positionIndicator);
```

Comment:
get used to adding a return type to functions. if the eslint rule it not set to that, update it.

```suggestion
    beforeDestroy = (): void => this.viewportUtil.removeResizeHandler(this.positionIndicator);
```